pkg/api/http_server.go

```diff
@@ -581,9 +581,9 @@ func (hs *HTTPServer) addMiddlewaresAndStaticRoutes() {
 	m := hs.web
 
 	m.Use(middleware.RequestTracing(hs.tracer))
-	m.Use(middleware.RequestMetrics(hs.Features))
 
 	m.UseMiddleware(middleware.Logger(hs.Cfg))
+	m.UseMiddleware(middleware.RequestMetrics(hs.Features))
```

Comment:
I'm curious why this is getting moved to after the logger middleware?

Reply:
Indeed! I had to make this change in order to make it work locally so I suspect I hit a bug that I haven't looked deeper into.